demo/utilities/data.ts

```diff
@@ -22,7 +22,7 @@ function hydrateGraph({ id, graph }: FlowRunGraphMock): FlowRunGraphMock {
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export function createDataStores(seeds: ApiMockSeeds = {}) {
   return {
-    artifacts: new KeyedDataStore({ seeds: seeds.artifacts, hydrate: artifact => new Artifact(artifact) }),
+    artifacts: new KeyedDataStore({ seeds: seeds.artifacts, hydrate: artifact => new Artifact({ ...artifact, created: new Date(artifact.created), updated: new Date(artifact.updated) }) }),
```

Comment:
This makes sense to me since dates in json don't get parsed automatically. But it makes me wonder how all the other dates in these mocks aren't breaking things 🤔

Reply:
Yeah this was surprising to me as well; it's possible we just aren't using any built in date methods anywhere and the strings we're passing around are perfectly fine for most of our date utilities